flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/OverWindowITCase.scala

```diff
@@ -2576,3 +2594,19 @@ class CountAggFunction extends AggregateFunction[JLong, CountAccumulator] {
 
   override def getResultType: TypeInformation[JLong] = Types.LONG
 }
+
+private class TestModule extends Module {
+
+  private val funcName = "isnull"
```

Comment:
`isnull` -> `isNull` ?

Reply:
I don't think that matters, because I'm using `equalsIgnoreCase` down bellow.